FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml

```diff
@@ -307,6 +320,9 @@
                                         ViewModel="{x:Bind KeyBindings}" />
                                 </controls:HeaderedContentControl>
 
+
+                                <TextBlock Margin="0,0,0,8" Text="Line ending translation on paste" />
```

Comment:
The way this is written is inconsistent with the other options, I think it should be more like this:

```suggestion
                                <controls:HeaderedContentControl
                                    Margin="0,36,0,0"
                                    HorizontalContentAlignment="Stretch"
                                    Header="Line ending translation on paste">
                                    <ContentControl
                                        Margin="0,6,0,0"
                                        Content="{x:Bind LineEndingStyle, Mode=OneWay, Converter={StaticResource EnumToDescriptionConverter}}"
                                        ContentTemplateSelector="{StaticResource TerminalThemeTemplateSelector}" />
                                </controls:HeaderedContentControl>
```